src/Service/Ses/tests/Unit/SesClientTest.php

```diff
@@ -15,13 +18,25 @@
 use AsyncAws\Ses\ValueObject\Message;
 use AsyncAws\Ses\ValueObject\RawMessage;
 use AsyncAws\Ses\ValueObject\Template;
-use PHPUnit\Framework\TestCase;
 use Symfony\Component\HttpClient\MockHttpClient;
 use Symfony\Component\HttpClient\Response\MockResponse;
 use Symfony\Contracts\HttpClient\ResponseInterface;
 
 class SesClientTest extends TestCase
 {
+    public function testGetSuppressedDestination(): void
+    {
+        $client = new SesClient([], new NullProvider(), new MockHttpClient());
+
+        $input = new GetSuppressedDestinationRequest([
+            'EmailAddress' => 'change me',
```

Comment:
It is a silly example, but we would prefer "real" values in case people read the tests as an example how to use it. 

```suggestion
            'EmailAddress' => 'test@example.org',
```